Fabric/src/main/resources/fabric.mod.json

```diff
@@ -55,6 +55,9 @@
     ],
     "rei_client": [
       "vazkii.botania.fabric.integration.rei.BotaniaREIPlugin"
+    ],
+    "jei_mod_plugin": [
+      "vazkii.botania.client.integration.jei.JEIBotaniaPlugin"
```

Comment:
is this actually necessary? I thought JEI uses annotations to identify plugins

Reply:
The annotation is only used on Forge. Since Fabric doesn't do jar scanning for annotations, JEI uses entrypoints there.